src/wp-includes/customize/class-wp-customize-nav-menu-location-control.php

```diff
@@ -56,6 +56,13 @@ public function render_content() {
 		if ( empty( $this->choices ) ) {
 			return;
 		}
+
+		$value_hidden_class = $no_value_hidden_class = '';
```

Comment:
Having this separate makes it more confusing. What was wrong with the if statement in the `button`?

Reply:
I comes from this change:
https://core.trac.wordpress.org/changeset/42217

It was necessary to backport this to fix some broken unit tests after the first pass.

Reply:
So it wasn't done in the same order as WP, where it was _before_ the auto-format?

Reply:
This PR is not a backport but an automated `phpcbf` correction for certain Coding Rules applied to our source, so there isn't really an order as such.
Before the additional backport of 42217 the `if` statement was inline in the HTML further down, for this particular file the diff is perhaps best seen here:
https://core.trac.wordpress.org/changeset/42217#file15

Reply:
I understand what this PR is. I'm trying to get whether you backported the preparation changeset before you did the auto format stuff, because that's what that changeset was for...preparing for auto format.
I think setting the variables so far away from where they are used is a bad thing, regardless of how it was done in WP. I would also argue that those changes were only needed for the auto format, and if they make the code harder to read, then don't do it.

Reply:
> It was necessary to backport this to fix some broken unit tests after the first pass.

Ah, I understand now, and as I said above, it only became apparent to me that the backport was needed after I'd run the `phpcbf` on the code, so in answer to your question then, no it wasn't done in order.

I suppose readability is a matter of opinion, and while this PR is about readability it is also about making future backports easier too. For the latter reason I would suggest we consider keeping these change in.

Reply:
As per review meeting, this is fine to keep the change